.env.sample

```diff
@@ -55,15 +56,17 @@ DJANGO_LOGGING_LEVEL=DEBUG
 # Default value for cache_page ttl
 # CACHE_TTL=600
 
-# Only supported package is Redis
+# Redis, Database LocMem and others are supported
 # If you don't want any cache set this to 
 # django.core.cache.backends.dummy.DummyCache 
 
-# This is the default if nothing is set
+# DummyCache is the default if nothing is set
 
+# Possible Options
 # CACHE_BACKEND=django_redis.cache.RedisCache
+# CACHE_BACKEND=django.core.cache.backends.db.DatabaseCache
 
-# Location of cache, for redis this is server and port, for Database it's table name
+# Location of cache, for redis this is server and port, for database it's table name
 # CACHE_LOCATION=redis://student_explorer_redis:6379/1
```

Comment:
this value will be used by "python manage.py createcachetable" to create cache table. Need a placeholder value here.

Reply:
1) I added additional details and reworked the sample configuration.

2) The start script is only checking to see if the Backend is "django.core.cache.backends.db.DatabaseCache" if it is it creates the cache table based on CACHE_LOCATION. Otherwise nothing is needed to be run. It's just an extra setup step for database caching.